src/pages/Pool/DeprecationBanner.tsx

```diff
@@ -0,0 +1,57 @@
+import { Link } from 'react-router-dom'
+import { Flex, Text } from 'rebass'
+
+import { APP_PATHS } from 'constants/index'
+import useTheme from 'hooks/useTheme'
+
+if (Date.now() > new Date('2023-08-12T00:00:00Z').getTime()) {
+  console.warn('Remove DeprecationBanner')
+}
```

Comment:
what is this? can we remove this?

Reply:
Just a reminder to remove this banner once it's expired